src/OpenTelemetry.Exporter.Jaeger/JaegerTraceExporter.cs

```diff
@@ -78,13 +82,5 @@ protected virtual void Dispose(bool disposing)
                 this.disposedValue = true;
             }
         }
-
-        private void ValidateOptions(JaegerExporterOptions options)
-        {
-            if (string.IsNullOrWhiteSpace(options.ServiceName))
-            {
-                throw new ArgumentException("Service Name is required", nameof(options.ServiceName));
```

Comment:
So this is a required parameter to the Process class ultimately right? Your change removes this check and just assumes that that service name will be part of the resource and if its not this is going to fail at runtime I think.

Reply:
@pcwiese Yes, this was a side effect of @lmolkova asking me to move the resource processing logic out of the construction phase. Final SeriviceName can no longer be known during ctor. How it is working as of last commit...

* When the first span comes in we apply the library resources and calculate the final service name + process tags. If ServiceName is null at this point, we use the default value "Open Telemetry Exporter" which is the same as ZipkinExporter.
* When first batch is appended JaegerUdpBatcher calculates the header size using the final Process ServiceName + Tags computed from the options + library resource.